### DISCUSSION TITLE
Welcome to numerical-veliz Discussions!

### DISCUSSION THREAD

**Discussion by osveliz** [osveliz/numerical-veliz#3] [General]
## 👋 Welcome!
  I am using Discussions as a place to connect with the community. Please use this space to:
  * Discuss videos
  * Suggest a topic
  * Report code issues
  * Share thoughts about numerical analysis and computer science

  To get started, comment below to share your thoughts with this community. Remember to follow the GitHub Community Guidelines.

**Comment by avachon100501** [comment]
Hello, Oscar!
The new Discussions feature is very promising.
And here are my thoughts about numerical analysis.

I was introduced to fractals back in the early 2010s (2012, maybe?), and the first fractal that I saw was the Mandelbrot set. Years later, numerical analysis and root-finding algorithms (such as Newton, Halley, Muller, Householder and Steffensen) allowed me to make more amazing fractals. I used very fast and intuitive software such as Kalles Fraktaler, FractInt, Ultra Fractal and ChaosPro to make explorations of such fractals.

With Ultra Fractal, and your videos, I progressed, and even managed to make multidimensional Newton fractals (so far I only did the two-dimensional variant), and I plan on doing Broyden's method, Jenkins-Traub and Lehmer-Schur. These methods may be a little bit more difficult, but it's always a fun challenge. Ultra Fractal also allows you to use different coloring algorithms, such as binary decomposition, exponential smoothing and triangle inequality average.

Here are fractals that I generated with Newton-Bisection and standard Newton's method for the function `z^3 - 2z + 2` for comparison.
![Newton-Bisection_3_1](https://user-images.githubusercontent.com/72226477/101854344-59248780-3b2f-11eb-983c-10cd4fb56f4e.png)
![NewtonGeneralized3](https://user-images.githubusercontent.com/72226477/101854346-59bd1e00-3b2f-11eb-90dd-d708732d3fe7.png)
More here: [https://github.com/avachon100501/ultrafractalcollection/blob/main/Root-Finding/Newton-Bisection%20Hybrid.ufm](url)
So, I think numerical analysis is very interesting to explore, and I like that. Keep up the good work!
